packages/perseus/src/widgets/interactive-graphs/graphs/screenreader-text.test.ts

```diff
@@ -0,0 +1,27 @@
+import {srFormatNumber} from "./screenreader-text";
+
+describe("srFormatNumber", () => {
+    it("trivially converts most integers to strings", () => {
```

Comment:
Nit: I think the wording here would be more accurate if you said "floats" or simply "numbers" in this description since decimal numbers are not integers.

Reply:
This description is just for the first test case, which tests `3`. I guess "most" integers are not like `3` (they are large and would have thousands separators).

That reminds me: I should add a test to ensure that scientific notation is not used.

Reply:
OH 🤦🏽 I don't know why I thought this was a comment above it rather than the description of the test. Sorry about that.